TramsDataApi/UseCases/GetAcademyConversionProject.cs

```diff
@@ -1,35 +1,32 @@
-﻿using TramsDataApi.DatabaseModels;
-using TramsDataApi.Factories;
+﻿using TramsDataApi.Factories;
 using TramsDataApi.Gateways;
-using TramsDataApi.RequestModels.AcademyConversionProject;
 using TramsDataApi.ResponseModels.AcademyConversionProject;
 
 namespace TramsDataApi.UseCases
 {
-    public class GetAcademyConversionProject : IUseCase<GetAcademyConversionProjectByIdRequest, AcademyConversionProjectResponse>
+    public class GetAcademyConversionProject : IGetAcademyConversionProject
     {
         private readonly IAcademyConversionProjectGateway _academyConversionProjectGateway;
-        private readonly ITrustGateway _trustGateway;
+        private readonly IEstablishmentGateway _establishmentGateway;
        
         public GetAcademyConversionProject(
             IAcademyConversionProjectGateway academyConversionProjectGateway,
-            ITrustGateway trustGateway)
+            IEstablishmentGateway establishmentGateway)
         {
             _academyConversionProjectGateway = academyConversionProjectGateway;
-            _trustGateway = trustGateway;           
+            _establishmentGateway = establishmentGateway;
         }
 
-        public AcademyConversionProjectResponse Execute(GetAcademyConversionProjectByIdRequest request)
+        public AcademyConversionProjectResponse Execute(int id)
         {
-            var academyConversionProject = _academyConversionProjectGateway.GetById(request.Id);
-            if (academyConversionProject == null) return null;
-
-            var trust = string.IsNullOrEmpty(academyConversionProject.TrustReferenceNumber)
-                ? null
-                : _trustGateway.GetIfdTrustByGroupId(academyConversionProject.TrustReferenceNumber);
-
-            return AcademyConversionProjectResponseFactory.Create(academyConversionProject, trust, null);
+            var academyConversionProject = _academyConversionProjectGateway.GetById(id);
+            if (academyConversionProject?.SchoolName == null) return null;
+            
```

Comment:
Do we need to think about what these schoolName == null filters are about and whether we want them?

Reply:
Will remove this, it appears that this was a workaround to ignore incomplete projects in an earlier iteration.  This should not happen with the new project creation approach